hamocc/mo_inventory_bgc.F90

```diff
@@ -827,7 +827,7 @@ subroutine write_netcdf(iogrp)
         !--- Define global attributes
         call nccheck(NF90_PUT_ATT(ncid,NF90_GLOBAL,'title','Global inventory for marine bgc') )
         call nccheck(NF90_PUT_ATT(ncid,NF90_GLOBAL,'history','Global inventory for marine bgc') )
-        call nccheck(NF90_PUT_ATT(ncid,NF90_GLOBAL,'date', timeunits) )
+        !call nccheck(NF90_PUT_ATT(ncid,NF90_GLOBAL,'date', timeunits) )
 
```

Comment:
Could be deleted instead of commented out?

Reply:
Done.